schema.sql

```diff
@@ -37,3 +37,26 @@ ADD COLUMN species_id INTEGER REFERENCES species(id);
 -- Add the owner_id column as a foreign key referencing the owners table
 ALTER TABLE animals
 ADD COLUMN owner_id INTEGER REFERENCES owners(id);
+
+
+CREATE TABLE vets (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255),
+    age INTEGER,
+    date_of_graduation DATE
+);
+
+CREATE TABLE specializations (
+    id SERIAL PRIMARY KEY,
+    vet_id INTEGER REFERENCES vets(id),
+    species_id INTEGER REFERENCES species(id)
+);
+
+
+CREATE TABLE visits (
+    id SERIAL PRIMARY KEY,
+    animal_id INTEGER REFERENCES animals(id),
+    vet_id INTEGER REFERENCES vets(id),
+    visit_date DATE
+);
+
```

Comment:
- [ ] The visits table is not created anytime I try to run this file. The reason is because of this statement in your SQL file:

```SQL
ALTER TABLE animals
DROP CONSTRAINT animals_pkey;
```
When you run this command, it will remove the primary key constraint from the `animals`. This means that the `id` column will no longer be a primary key, and you can't subsequently create a new foreign key referencing it. 

I would advise you to remove this statement from your SQL file 👍 😎 

![image](https://github.com/Aqurds/sql-schema/assets/77106962/ef69fa03-b3c0-405f-908b-59b84986e767)